app/controllers/mobile_phone_controller.rb

```diff
@@ -28,7 +28,8 @@ def verify_token
                                                   params[:token])
 
     if errors
-      handle_errors(errors, "Failed to verify code to #{number}")
+
+      handle_errors([], "**Verifying phone number**")
```

Comment:
Ah sorry, I think there's a misunderstanding here.  When [@tdg wrote `**Verifying phone number**`](https://github.com/swapmyvote/swapmyvote/issues/131#issuecomment-557829090) I think he was referring to the *existing* `<h1>` header.  We don't want this text to appear again above it.